external/CMakeLists.txt

```
@@ -325,6 +325,10 @@ target_include_directories(libatrac9 PUBLIC LibAtrac9/C/src)
 
 option(BUILD_SHARED_LIBS "Build shared library" OFF)
 option(XXHASH_BUILD_XXHSUM "Build the xxhsum binary" OFF)
+option(XXH_X86DISPATCH_ALLOW_AVX "Allow building XXH3 with AVX even if it crashes on SSE2-Only CPUs" OFF)
+if(XXHASH_BUILD_XXHSUM)
+	add_definitions(-DXXHASH_BUILD_XXHSUM)
+endif()
 set(DISPATCH 1)
 add_subdirectory(xxHash/cmake_unofficial EXCLUDE_FROM_ALL)
```

```suggestion
set(DISPATCH 1)
add_subdirectory(xxHash/cmake_unofficial EXCLUDE_FROM_ALL)
if(XXH_X86DISPATCH_ALLOW_AVX)
    target_compile_definition(xxhash PRIVATE XXH_X86DISPATCH_ALLOW_AVX)
 endif()
```

What does XXHASH_BUILD_XXHSUM have to do with AVX?

---

oh my god i forgot to save the file im so dumb

---

should be good now